pkg/operator/api/api.go

```diff
@@ -26,12 +26,31 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/client-go/pkg/api/v1"
 	extensions "k8s.io/client-go/pkg/apis/extensions/v1beta1"
+	"k8s.io/client-go/pkg/apis/rbac/v1beta1"
 )
 
 const (
 	DeploymentName = "rook-api"
 )
 
+var clusterAccessRules = []v1beta1.PolicyRule{
```

Comment:
the permissions required will need to be updated for many reasons in the future so i don't think we need a special TODO comment